client/app/components/form/widgets/MultiSelectWidget.tsx

```diff
@@ -0,0 +1,103 @@
+"use client";
+
+import { Autocomplete, Chip, MenuItem, TextField } from "@mui/material";
+import { WidgetProps } from "@rjsf/utils/lib/types";
+import { DARK_GREY_BG_COLOR, BC_GOV_SEMANTICS_RED } from "@/app/styles/colors";
+
+interface Option {
+  const: string | number;
+  title: string | number;
```

Comment:
`const` and `title` are both key names in RJSF schemas, so I'd suggest changing these to something else to avoid mix-ups (also I don't know what they are just from the key name--const is the id maybe, and title is the text we want to show?)

Reply:
Thanks, I have change the values to `id` and `label` which seems more clear.